ecosystem/sep-0010.md

```diff
@@ -262,12 +277,6 @@ This convention ensures that SEP-10 clients and servers can use the same passphr
 
 The client can examine the `network_passphrase` (if defined) that the server includes in its response from the challenge endpoint to be sure it's using the correct passphrase and is connecting to the server that it expected.
 
-## JWT best practices
```

Comment:
Could we do the JWT expiration/best practices change in a separate PR? It seems unrelated to the rest of this change.

Reply:
It is unrelated but not meaningful, just combines the two JWT sections since the `JWT best practices` was one sentence.

Reply:
I can undo the change if we don't want to include it